servicetalk-data-jackson-jersey3-jakarta10/build.gradle

```diff
@@ -68,6 +68,13 @@ tasks.sourcesJar.dependsOn(copySourcesForJersey3)
 tasks.compileJava.dependsOn(copySourcesForJersey3)
 tasks.processTestResources.dependsOn(copySourcesForJersey3)
 tasks.compileTestJava.dependsOn(copySourcesForJersey3)
+afterEvaluate {
+  if (tasks.findByName("projectHealth")) {
+    tasks.withType(com.autonomousapps.tasks.CodeSourceExploderTask) {
```

Comment:
I tried many different ways to avoid this duplication. Unfortunately, nothing worked without bringing more problems. My gradle-foo is exhausted at this point 😢